### DISCUSSION TITLE
Not scrobbling to lastfm?

### DISCUSSION THREAD

**Discussion by benjamindaines** [navidrome/navidrome#3507] [General]
I am using Symfonium as a client, and it doesn't appear to be scrobbling to lastfm despite having lastfm enabled and the appropriate api keys programmed into the config file.  I can see the player in the navi webui, and send scrobbles to external is enabled... but nothing is updated on my lastfm page.

**Comment by kgarner7** [comment]
Change the log level to DEBUG, and see if you see any DEBUG logs in the form "Buffering Scrobble...", "Sending scrobble..." (these messages show scrobbling to an external service). Also check that you see info logs in the form "msg=Scrobbled" (this means that at least the client is sending scrobbles to Navidrome) and make sure that you have linked Last.FM in the user page (User icon > personal).

To verify that scrobbling is set up, if you SQL, you can use sqlite to run the following
```sql
SELECT user.name, value from user_props JOIN user ON user.id = user_id WHERE key = 'LastFMSessionKey';
```
If this shows nothing, then you have not setup Lastfm scrobbling correctly.

**Reply by benjamindaines** [reply]
> make sure that you have linked Last.FM in the user page (User icon > personal).

Thanks, that fixed it.  Not sure at what point that setting appeared (maybe after I rebooted the whole system rather than just restarted the service? idk), because I know for sure there was only an option to scrobble to musicbrainz.  Either way, it's enabled now and scrobbling properly :)

**Reply by kgarner7** [reply]
I've heard people reporting that they have needed to clear the cache (or open in private mode) once the properties are set properly. That's _likely_ what happened to you

**Reply by benjamindaines** [reply]
I think that's exactly what's happening.  I just noticed that the setting had, once again, disappeared.  I just tried in a private window and, yup, there it is.

**Reply by dwightyy** [reply]
Just wanted to report that I had the same problem on a fresh install. Scrobble to lastfm was not showing, only scrobble to listenbrainz.
I swapped to a private tab logged in and the button appread, I logged in and worked perfectly.

image: deluan/navidrome:latest on docker with portainer on local connection.
